app/src/main/java/com/miguelgaeta/android_media_picker/AppActivity.java

```diff
@@ -111,4 +130,23 @@ public void onCancelled() {
     public Context getContext() {
         return this;
     }
+
+    private class MyContentFileProvider implements MediaPicker.ContentFileProvider {
```

Comment:
Given that creating a file and the `FileProvider` are tightly coupled via the filepath, I don't think we should provide a default implementation.